src/nupic/research/spatial_pooler.py

```diff
@@ -325,8 +332,9 @@ def __init__(self,
 
     # Initialize a tiny random tie breaker. This is used to determine winning
     # columns where the overlaps are identical.
-    self._tieBreaker = 0.01*numpy.array([self._random.getReal64() for i in
```

Comment:
I believe the current (0.01xRandom) tie-breaker value is too significant and shapes the SDR, while it should be left to the column inhibition process and this array would really only break ties - a value around the permanence's precision would suffice. What do you think?